bucket/cursor.json

```diff
@@ -1,23 +1,30 @@
 {
-    "version": "0.45.14",
+    "version": "0.48.8",
     "description": "The AI Code Editor",
     "homepage": "https://www.cursor.com/",
     "license": {
         "identifier": "Proprietary",
         "url": "https://www.cursor.com/terms-of-service"
     },
+    "depends": "innounp",

```

Comment:
Can it be replaced to use `"innosetup": true`?

Reply:
No, if the replacement is made, the following error will occur.
![image](https://github.com/user-attachments/assets/0e9c0b59-8e03-4c78-a9f7-1febbaf32e42)

Reply:
Just use:

```json
    "innosetup": true,
    "extract_dir": "{code_GetDestDir}",
```

The tools dir is not needed imo.

Reply:
> Just use:
> 
> ```json
>     "innosetup": true,
>     "extract_dir": "{code_GetDestDir}",
> ```
> 
> The tools dir is not needed imo.

Got！You are right!